conf/AutoBalance.conf.dist

```diff
@@ -757,15 +760,39 @@ AutoBalance.reward.dungeonToken = 47241
 #       Default:     1
 AutoBalance.reward.MinPlayerReward = 1
 
-# The following variables are deprecated and should not be used in new deployments.
+##########################
+#
+# Announcement
+#
+##########################
+
+#
+#     AutoBalanceAnnounce.enable
+#        Announce the module on login if it is enabled
+#        Default:     1 (1 = ON, 0 = OFF)
+AutoBalanceAnnounce.enable=1
+
+##########################
+#
+# Deprecated Settings
+#
+##########################
+
+# The following variables are deprecated and should not be used in new deployments. Their values should be left blank.
 # They will still be applied to support backwards compatability, but will be removed entirely in a future release.
 # Their entire functionality (and more) has been moved to new settings referenced in this config file.
 #
-#   AutoBalance.PerDungeonScaling
-#   AutoBalance.PerDungeonBossScaling
-#   AutoBalance.BossInflectionMult
-#   AutoBalance.rate.global
-#   AutoBalance.rate.health
-#   AutoBalance.rate.mana
-#   AutoBalance.rate.armor
-#   AutoBalance.rate.damage
+AutoBalance.enable=
```

Comment:
put the default value instead of nothing, it should be better

Reply:
I may misunderstand how this works, but I think I need to leave it like this. This is a deprecated setting that is replaced by `AutoBalance.Enable.Global`. I still take in the old setting to maintain functionality, but I emit a warning if the user has the deprecated value in their config.

If I don't include the variable at all in the `.conf.dist` file and the user's config includes it, the variable doesn't get read and you get an error during server startup from the loader (not from my module). That doesn't really direct the user as to what to do about it, and it doesn't honor their setting of 0 or 1.

If I leave the setting blank as I have it here and the user has an old config with the deprecated variable, the value in the user's config still gets loaded and I emit the deprecation warning for them to address.

If I set it to 1 as you suggest, the user wouldn't be notified if they have the deprecated setting, and I would also have a problem with determining if the setting is leftover from an old config or what I put in the `conf.dist` file.

This is the solution I came up with that meets all the situations I can think of. Should I be doing something else?

Reply:
ok